poc/vectors/curve448_XOF:SHAKE256_ELL2_NU_.json

```diff
@@ -0,0 +1,90 @@
+{
+  "L": "0x54",
+  "Z": "0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffffffffffffffffffffffffffffffffffffffffffffffffffffe",
+  "ciphersuite": "curve448_XOF:SHAKE256_ELL2_NU_",
+  "curve": "curve448",
+  "dst": "QUUX-V01-CS02-with-curve448_XOF:SHAKE256_ELL2_NU_",
+  "expand": "XMD",
```

Comment:
```suggestion
  "expand": "XOF",
```

Reply:
Bug in the suite generation code. Will fix.

Reply:
@armfazh done.